src/iiiftilesource.js

```diff
@@ -239,10 +248,15 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
                         $.console.error('Data has a @context property which contains no known IIIF context URI.');
                 }
             }
-            if ( !data['@id'] && data['id'] ) {
-                data['@id'] = data['id'];
+
+            // normalize the various possible id properties to an internal one
+            data._id = data["@id"] || data["id"] || data['identifier'] || null;
```

Comment:
We need to not have this normalization here, because this is just changing the data that's going into the constructor, and the constructor doesn't expect `_id` (and it shouldn't).